internal/backends/selector/api.rs

```diff
@@ -111,6 +125,51 @@ impl BackendSelector {
         self
     }
 
+    /// Configures this builder to use the specified winit hook that will be called before a Window is created.
+    ///
+    /// It can be used to adjust settings of window that will be created.
+    ///
+    /// # Example
+    ///
+    /// ```rust,no_run
+    /// let mut backend = slint::BackendSelector::new()
+    ///     .with_winit+window_attributes_hook(|attributes| attributes.with_content_protected(true))
+    ///     .select()
+    ///     .unwrap();
+    /// ```
+    ///
+    /// *Note*: This function is behind a feature flag and may be removed or changed in future minor releases,
+    ///         as new major Winit releases become available.
+    #[must_use]
+    #[cfg(feature = "unstable-winit-030")]
+    pub fn with_winit_030_window_attributes_hook(
+        mut self,
+        hook: impl Fn(
+                i_slint_backend_winit::winit::window::WindowAttributes,
+            ) -> i_slint_backend_winit::winit::window::WindowAttributes
+            + 'static,
+    ) -> Self {
+        self.winit_window_attributes_hook = Some(Box::new(hook));
+        self
+    }
+
+    /// Configures this builder to use the specified winit event loop builder when creating the event
+    /// loop.
+    ///
+    /// *Note*: This function is behind a feature flag and may be removed or changed in future minor releases,
+    ///         as new major Winit releases become available.
+    #[must_use]
+    #[cfg(feature = "unstable-winit-030")]
+    pub fn with_winit_030_event_loop_builder(
+        mut self,
+        event_loop_builder: i_slint_backend_winit::winit::event_loop::EventLoopBuilder<
+            i_slint_backend_winit::SlintUserEvent,
```

Comment:
We also need to re-export SlintUserEvent

i guess SlintEvent is a better name?

We could also have a re-exported `type i_slint_backend_winit::EventLoopBuilder = winit::EventLoopBuilder<SlintEvent>`